src/Build/Evaluation/LazyItemEvaluator.IncludeOperation.cs

```diff
@@ -104,7 +104,7 @@ protected override ImmutableArray<I> SelectItems(OrderedItemDataCollection.Build
                             string[] includeSplitFilesEscaped;
                             if (MSBuildEventSource.Log.IsEnabled())
                             {
-                                MSBuildEventSource.Log.ExpandGlobStart(_rootDirectory, glob, string.Join(", ", excludePatternsForGlobs));
+                                MSBuildEventSource.Log.ExpandGlobStart(_rootDirectory ?? string.Empty, glob, string.Join(", ", excludePatternsForGlobs));
```

Comment:
I think this is verifiably not null, though if the full path does not contain any directory information (not entirely clear on how this could be, but it's in a comment), it may be empty. Not sure if that's a meaningful distinction you care to capture.